plugins/parsers/csv/parser.go

```diff
@@ -259,9 +382,22 @@ outer:
 
 	// add default tags
 	for k, v := range p.DefaultTags {
+		_, exists := tags[k]
+		if exists {
+			p.Log.Warnf("Tag %s has been overwritten.", k)
+		}
```

Comment:
What is this good for? That's the meaning of a _default_ IMO to be overwritten if you have other evidence. Please remove it as it has the potential to spam log-files for large number of similarly structured metrics.

Reply:
It comes because he is overwriting the value with a 'default'. So other way round and informing user about that is a good thing IMHO.